### PR TITLE
CON-3190 Fix projects structure in `AI` branch

### PR DESCRIPTION
**Pull request by Oumaimafisaoui**
Resolves. CON-3190

It fixes this issue as well: [Emotions detector issue](https://github.com/01-edu/public/issues/2746)

### CODE REVIEW

File: subjects/ai/emotions-detector/README.md [hunk 1]
```diff
@@ -148,14 +158,16 @@ Preprocessing ...
...
 ```
 
-### Useful resources:
+### Tips
+
+Balance technical prowess with psychological insight: as you fine-tune your CNN and optimize your video processing, remember that understanding the nuances of human facial expressions is key to creating a truly effective emotion detection system.
+
+### Resources
 
 - https://machinelearningmastery.com/what-is-computer-vision/
```
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/ai/emotions-detector/README.md:167]: Can you please further clarify what do you mean?

File: subjects/ai/credit-scoring/README.md [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
+#### Scoring model
 
 There are 3 expected deliverables associated with the scoring model:
 
 - An exploratory data analysis notebook that describes the insights you find out in the data set.
 - The trained machine learning model with the features engineering pipeline:
 
   - Do not forget: **Coming up with features is difficult, time-consuming, requires expert knowledge. ‘Applied machine learning’ is basically feature engineering.**
-  - The model is validated if the **AUC on the test set is higher than 75%**.
+  - The model is validated if the **AUC on the test set is higher than 50%**.
```
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/ai/credit-scoring/README.md]: It was updated to be between 55% to 62%, the capacity of the devices was taken into consideration.

File: subjects/ai/emotions-detector/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/ai/emotions-detector/README.md]: done.

File: subjects/ai/nlp-scraper/README.md [hunk 3]
```diff
@@ -127,7 +145,7 @@ Here are examples of insights:
   - Companies mentioned the most
   - Sentiment per companies
 
-### Deliverables
+### Project repository structure:
```
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/ai/nlp-scraper/README.md:148]: fixed.

File: subjects/ai/credit-scoring/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zanninso) [reply subjects/ai/credit-scoring/README.md]: it must be one specific value and not a range it can be more then 55% no need to specify 62% , and i really don't understand what is the relation between that and the devices capacity! as i know it about the model and how it trained

**Review by zanninso — APPROVED** [review]
LGMT!
